ajax/libs/primer-markdown/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "primer-markdown",
+  "description": "GitHub stylesheets for rendering markdown.",
+  "filename": "build/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/primer/primer/tree/master/modules/primer-markdown"
+  },
+  "author": {
+    "name": "GitHub, Inc."
+  },
+  "keywords": [
+    "github",
+    "markdown",
+    "css",
+    "scss",
+    "primer",
+    "style",
+    "stylesheets",
+    "md"
+  ],
+  "homepage": "https://github.com/primer/primer-markdown",
```

Comment:
If the homepage is GitHub repository, then we don't add this field.
Please remove this line.
Thanks.